[2022-06-27 13:30:22,204] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-02T00:00:00+00:00 [queued]>
[2022-06-27 13:30:22,255] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-02T00:00:00+00:00 [queued]>
[2022-06-27 13:30:22,257] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:22,261] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:30:22,263] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:22,696] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-01-02 00:00:00+00:00
[2022-06-27 13:30:22,777] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-01-02T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpp53xrpc3', '--error-file', '/tmp/tmp31bkyw5t']
[2022-06-27 13:30:22,780] {standard_task_runner.py:80} INFO - Job 17: Subtask model_A
[2022-06-27 13:30:22,766] {standard_task_runner.py:52} INFO - Started process 56629 to run task
[2022-06-27 13:30:23,504] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-01-02T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:30:24,139] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T00:00:00+00:00
[2022-06-27 13:30:24,154] {python.py:173} INFO - Done. Returned value was: 3
[2022-06-27 13:30:24,459] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220102T000000, start_date=20220627T133022, end_date=20220627T133024
[2022-06-27 13:30:24,552] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:30:25,071] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
